src/lib/platform/XWindowsScreen.cpp

```diff
@@ -1548,6 +1548,11 @@ XWindowsScreen::onMouseMove(const XMotionEvent& xmotion)
 {
 	LOG((CLOG_DEBUG2 "event: MotionNotify %d,%d", xmotion.x_root, xmotion.y_root));
 
+
+	// only consider motion on the same X screen as we were started on
```

Comment:
Can we use spaces for all new code?

Reply:
I can, but can I request you guys do a mass formatting change to convert from tabs to spaces?  Is already a bit annoying when editing the code because indenting isn't lining up having it mixed.